css/easyzoom.css

```diff
@@ -46,6 +46,22 @@
 	background: #FFF;
 }
 
+.easyzoom-lens {
+	position: absolute;
+	top: 0;
+	left: 0;
```

Comment:
The `top` and `left` properties can be removed as they're specified in the JS anyway